ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/project/node/ProjectDescriptorNode.java

```diff
@@ -63,6 +58,13 @@ public void updatePresentation(@NotNull NodePresentation presentation) {
         presentation.setPresentableText(getData().getName());
         presentation.setPresentableIcon(isValid(getData()) ? nodeManager.getNodesResources().projectFolder()
                                                            : nodeManager.getNodesResources().notValidProjectFolder());
+        presentation.setPresentableTextCss("font-weight:bold");
+
+        if ("private".equals(getData().getVisibility())) {
```

Comment:
It's an experemental behaviour. If we'll decide to show some "unlock icon" for public projects it won't be difficult to add it.